prog/setup_spdk.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Prog::SetupSpdk < Prog::Base
+  subject_is :sshable, :vm_host
+
+  def start
+    fail "Not enough hugepages" unless vm_host.total_hugepages_1g > vm_host.used_hugepages_1g
+    sshable.cmd("sudo bin/setup-spdk")
+    sshable.cmd("sudo systemctl start spdk")
```

Comment:
Separating them to different steps might increase robustness.

Currently, if `bin/setup-spdk` is succeeded and `systemctl start spdk` is failed, it will try to setup it again instead of just try to start in next run.

Reply:
Addressed in https://github.com/ubicloud/ubicloud/pull/212